lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -178,12 +201,69 @@ def _install_internal_conntrack_flow(self):
                                              reset_default_register=False,
                                              resubmit_table=self.config.internal_conntrack_fwd_tbl)
 
+    def _send_messsage_interface(self, ue_ip_add:str):
+        """
+        Build the message
+        """
+        ofproto, parser = self._datapath.ofproto, self._datapath.ofproto_parser
+        match = MagmaMatch(eth_type=ether_types.ETH_TYPE_IP, ipv4_src=ue_ip_add)
+        ryu_match = parser.OFPMatch(**match.ryu_match)
+        req = parser.OFPFlowStatsRequest(self._datapath, table_id=14,
+                out_group=ofproto.OFPG_ANY,
+                out_port=ofproto.OFPP_ANY,match=ryu_match)
+        try:
+            messages.send_msg(self._datapath, req)
+            self.pagingflag = 1
+        except MagmaOFError as e:
+            self.logger.warning("Couldn't poll datapath stats: %s", e)
+            self.pagingflag = 0
+
+    @set_ev_cls(ofp_event.EventOFPFlowStatsReply, MAIN_DISPATCHER)
+    def _flow_info_reply_handler(self, ev):
+        if self.pagingflag:
+            flow_info = ev.msg.body
+            self.pagingflag = 0
+            if ev.msg.flags == OFPMPF_REPLY_MORE:
+                # Wait for more multi-part responses thats received for the
+                # single stats request.
+                return
+            if flow_info:
+                self.loop.call_soon_threadsafe(
+                                       self._handle_flow_info, flow_info)
+
+    def _handle_flow_info(self, flow_info):
+        for stat in flow_info:
+            sid = _get_sid(stat)
+            ipv4_addr = _get_ipv4(stat)
+            tunnel_id = _get_tunnel_id(stat)
+
+        if (sid != None and ipv4_addr !=None and tunnel_id):
+            paging_message=UPFPagingInfo(subscriber_id = sid, local_f_teid=tunnel_id,
+                                         ue_ip_addr=ipv4_addr)
+
+            future = self._sessiond_setinterface.SendPagingReuest.future(
+                            paging_message, self.SESSIOND_RPC_TIMEOUT)
+            future.add_done_callback(
+                              lambda future: self.loop.call_soon_threadsafe(
+                              self._callback_method, future))
+
+    def _callback_method(self, future):
```

Comment:
can you name the callback better something like paging_msg_sent_callback

Reply:
Yes. Name changed.